src/Microsoft.TemplateEngine.TemplateLocalizer/LocalizableStrings.resx

```diff
@@ -144,7 +144,12 @@
     <comment>{0} is the number of files. Do not localize "--export"</comment>
   </data>
   <data name="command_export_log_templateExportFailedWithError" xml:space="preserve">
-    <value>Generating localization files for the following template.json file has failed: "{0}". Error message: "{1}".</value>
+    <value>Generating localization files for a template.json has failed.
+Reason: {0}.
+File Path: {1}.</value>
+    <comment>{0} is a sentence such as "Json property should contain a field 'ID'"
```

Comment:
previously we discussed, that if 'id' is missing it would be generated by localizer. Have we dropped this feature?

Reply:
Are you sure about the discussions? Localizer doesn't make changes to the template.json file. And I don't recall that we came to such a conclusion.

Reply:
Maybe I misunderstood it. 
I thought that if ids are not present the tool will auto-generates them and adds them to `template.json` and `templatestrings`. Imo this will be helpful, as the author won't need to create ids manually. Also it will allow to cover cases of multiple entries, as we can do ids similar to "autogen_1", "autogen_2".